app/controllers/carts_controller.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class CartsController < ApplicationController
   before_action :intialize_Cart_Object, only: %i[show destroy]
```

Comment:
@amitshethe update method name in snake_case and where is show action in this controller?